visualize_decisions.py

```diff
@@ -0,0 +1,90 @@
+"""
+Gregory Way 2017
+PanCancer Classifier
+visualize_decisions.py
+
+Usage: Run in command line with required command argument:
+
+        python visualize_decisions.py --scores $prediction_file
+
+Where prediction_file is a string pointing to the location of sample scores
+
+Output:
+.tsv file of classifier scores and other covariate info for plotting
+"""
+
+import os
+import argparse
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+parser = argparse.ArgumentParser()
```

Comment:
This is really a minor point. Just wondering whether line 22-27 should go after the function definition.